osu.Game.Rulesets.Osu/Mods/OsuModClassic.cs

```diff
@@ -17,7 +17,7 @@ namespace osu.Game.Rulesets.Osu.Mods
 {
     public class OsuModClassic : ModClassic, IApplicableToHitObject, IApplicableToDrawableHitObject, IApplicableToDrawableRuleset<OsuHitObject>
     {
-        public override Type[] IncompatibleMods => new[] { typeof(OsuModStrictTracking) };
+        public override Type[] IncompatibleMods => base.IncompatibleMods.Append(typeof(OsuModStrictTracking)).ToArray();
```

Comment:
This change has now been rendered sort of unrelated to the rest of the diff, but it's not *wrong*, so...